### DISCUSSION TITLE
Bokeh Inline Style Sheets Keys/Tables

### DISCUSSION THREAD

**Discussion by Nmirabile777** [bokeh/bokeh#14010] [Documentation]
Hello,

I have been using Bokeh for a while and have been loving it. I am now starting to move on to more advanced formatting requiring CSS using InlineStyleSheets. I have been searching for a documentation source to reference certain classes such as the tabs in Bokeh being .bk-tab.bk-active for example. Is there a central location to find information regarding this so I can apply CSS to individual and specific Bokeh elements?

Thanks,
Nick

**Comment by mattpap** [comment]
No, at this point there is no such place either in the code base or in documentation. We are currently in the long term process of improving the styling situation in bokeh, which includes adding a well defined set of CSS classes and variables to all bokeh's components. This should happen either in bokeh 3.6 or 3.7, depending on the progress.

**Reply by Nmirabile777** [reply]
Hi Matt,

Thank you for the response, understood and looking forward to using Bokeh more!

Thanks,
Nick
